content/docs/overview/about.md

```diff
@@ -0,0 +1,95 @@
+
+{{< picture src="overview.png" alt="NuoDB Control Plane overview" >}}
+
+## Key concepts
+
+The NuoDB Control Plane can be broken down into several communicating layers.
+
+{{< picture src="cp-arch-layers.png" alt="NuoDB Control Plane logical architecture" >}}
+
+### REST Service
+
+DBaaS REST Service allows multi-tenant access for external customers according to access control rules.
+It exposes a REST API that enables remote management of NuoDB domains and databases using coarse-grained CRUD operations (Create, Read, Update, and Delete).
+The data model used by the DBaaS REST Service to define user access control is summarized below:
+
+- _Organizations_ have several users and policies associated with them.
+- _Projects_ are logical groupings of databases and have a service tier associated with them.
+- _Users_ have roles that control access to projects and databases.
+
+Projects abstract the aspects of domains that are relevant for DBaaS users and provide isolation between databases.
+Databases within the same project will inherit the service tier from the project to which they belong, and databases in different projects will be part of different domains.
+
+The REST APIs for projects and databases internally manipulate Domain and Database custom resources (CRs), respectively.
+
+### DBaaS Operator
+
+The DBaaS Operator is a [Kubernetes Operator](https://coreos.com/operators/) which deploys NuoDB into the Kubernetes cluster.
+The Go Operator enables easy deployment of the NuoDB domain and database into the local Kubernetes cluster by using [NuoDB Helm Charts](https://github.com/nuodb/nuodb-helm-charts) and exposing high-level configuration to the user.
+
+### NuoDB Helm charts
+
+The NuoDB Control Plane uses the production-ready NuoDB _admin_ and _database_ Helm charts to deploy a domain (Admin Processes, APs) and a database (TEs and SMs) respectively.
+
+### Kubernetes
+
+Ultimately everything runs in a Kubernetes cluster - the NuoDB Control Plane itself and all APs, TEs and SMs it deploys.
+Configuration information is held in _ConfigMap_ and _Secret_ resources.
+
+### DBaaS Configuration and Offerings
+
+Internally the NuoDB Control plane uses _Service Tiers_ and _Helm Features_ to simplify database deployment and expose various predefined configuration options available to DBaaS end users.
+See [Service Tiers]({{< ref "../administration/offerings/service-tiers.md" >}}), for more information on creating re-usable configuration options.
+
+### NuoDB
+
+NuoDB is a distributed RDBMS that runs as a multi-process, distributed architecture running on multiple hosts, in multiple data centers.
```

Comment:
Suggest: 

NuoDB is a distributed Relational Database Management System (RDBMS) that operates as a multi-process, distributed architecture across multiple hosts and data centers. NuoDB is composed of TE and SM database processes, which are managed by APs. For more information on NuoDB's system architecture, refer to the NuoDB System Architecture documentation.

(also, suggest spelling out the use of AP, SM, and TE terms during their first usage on the page, then use the acronyms thereafter.)